_projects/area_indicators.md

```diff
@@ -0,0 +1,66 @@
+---
+published: false # change to true once ready to post
+
+title: "Climate Change API Area Indicators" # project title inside quotes
+excerpt: "A high performance Scala API for calculating climate indicators over geospatial areas" # shows on project list page
+
+# project attributes
+requirements: # bullet list of requirements – one requirement per line – follow below format
+  - "Familiarity building or working with APIs"
+  - "Interest and/or exerience in functional programming required (Haskell, Scala, ML, etc.)"
+  - "Scala programming experience not required but preferred"
+  - "Python programming experience not required but preferred"
+tags: # one tag per line – spaces are allowed in tags – follow below format
+  - "scala"
+  - "climate change"
+difficulty: "Medium" # easy, medium, hard – pick one
+mentors: # github username without the @ – example: designmatty
+  - "maurizi"
+  - "pcaisse"
+  - "jamesmcclain"
+  - "jisantuc"
+  - "fungjj92"
```

Comment:
This mentor list is long - it would be great to have only 2-3 on a team. One lead mentor, one secondary mentor, and potentially someone who had knowledge of the projects or technology that could serve as a resource. Having 5 members will make it difficult to schedule screening and interview sessions and might be confusing to potential applicants around who would actually be mentoring.

Can you whittle the list down to the folk who will make up the 2 person mentor team/support, with one additional mentor or project resource person?

Reply:
I removed @jamesmcclain and myself from the mentor list.